iOS internship/ViewController.swift

```diff
@@ -45,13 +40,10 @@ class ViewController: UIViewController, UITableViewDataSource, UITableViewDelega
         }
     }
     
-    //    loadedData(compelte: () -> () ) {
-    //
-    //    }
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         
-        let cell = UITableViewCell(style: UITableViewCell.CellStyle.default, reuseIdentifier: nil)
+        let cell = UITableViewCell(style: UITableViewCell.CellStyle.default, reuseIdentifier: "Cell")
```

Comment:
Give cell a meaningful identifier, it should be obvious what that cell holds from looking at the identifier.